lms/djangoapps/discussion/tasks.py

```diff
@@ -90,3 +93,26 @@ def _get_thread_url(context):
         'id': context['thread_id'],
     }
     return urljoin(settings.LMS_ROOT_URL, permalink(thread_content))
+
+
+def _generate_ga_pixel_url(course_key, user_id):
+    # used for analytics
+    query_params = {
```

Comment:
It would be great if you could include comments next to each of these parameters with a human readable description of them?

Reply:
Will do!